docs/content/docs/dev/troubleshooting.md

```diff
@@ -32,14 +32,100 @@ We are using [`path_provider`'s `getApplicationDirectory`](https://pub.dev/packa
 Just opening a terminal and doing `rm -rf $PATH` (where `$PATH` is the necessary path above), usually resets all locally stored data. **Careful as this also removes all locally held crypto tokens**.
 
 ## Execution failed for task ':app:processDebugMainManifest'.
```

Comment:
This error shouldn't happen now with flutter latest version. We can remove this doc part or direct users to update to latest..

Reply:
this was actually from a previous PR that is now not part of this PR anymore as it is already in `main`... If you still want to discuss that, let's address it separately.